packages/kuma-gui/src/app/x/index.ts

```diff
@@ -101,6 +103,7 @@ export const services = (app: Record<string, Token>): ServiceDefinition[] => {
           ['XTeleportSlot', XTeleportSlot],
           ['XTimespan', XTimespan],
           ['XDisclosure', XDisclosure],
+          ['XAboutSection', XAboutSection],
```

Comment:
⚠️  Large warning: naming discussion ⚠️  😅 

We have an opportunity to redesign this interface (inc the name) here.

Do you think its weird to name a component after a place that you put it, especially when you could put it anywhere? Would it be better to name it after what it is? To me this component is an `XCard` that has a `Timespan` (or just `Date`).

i.e. `XDateCard`, which happens to fit with other components we have such as `DefinitionCard`, `DataCard` etc, and means our "generic component set" feels the same throughout all its components (similar to using say `:variant` whenever we have a type-like setting)

tiny thing, but I think we are going to make a a few changes here anyway, wdyt?

Reply:
That's a very good point 🤔 I think there are some (very rare) cases in which I would name the component based on the location and only if the component is not meant to be placed anywhere else (still more than one place in the app). So, in this case I'd be fine either way and I do see benefits in both approaches:

1. Name the component based on the location: should not be used anywhere else and the name does not require any changes in the future. This means we are flexible in how the component evolves, but not flexible in where we can put it. The downside is that if we need to add this component anywhere else, we might create a similar one or come back to this discussion.
2. Name the component based on what it is: allows to put the component anywhere and the name probably needs to change if the content of the component changes (e.g. at some point we add a description text). This means we are flexible in where we put the component, but probably not so flexible in how the component evolves. Also here, we would probably come back to this discussion when the component evolves.

Reply:
I left a comment here about another potential name - `XAboutCard`, I think it coinciding with a GH blip tho. Not sure from your comment whether we are looking to rename or not, lmk